modules/javafx.web/src/main/native/Source/WebCore/platform/java/ChromeClientJava.cpp

```diff
@@ -648,8 +648,7 @@ void ChromeClientJava::reachedApplicationCacheOriginQuota(SecurityOrigin&, int64
     notImplemented();
 }
 
-#if USE(ACCELERATED_COMPOSITING)
```

Comment:
shouldn't this still be conditional, e.g. for platforms that don't support the required OpenGL extensions?

Reply:
In the recent WebKit, ACCELERATED_COMPOSITING preprocessor define is removed and they made it as a runtime feature. In this patch, ACCELERATED_COMPOSITING enabled based on [ConditionalFeature.SCENE3D runtime flag](https://github.com/javafxports/openjdk-jfx/pull/51/commits/aac6b9841c9c48df685edab22abd19814dc68311).